tidb-cloud/tidb-cloud-console-auditing.md

```diff
@@ -0,0 +1,150 @@
+| AuditEventUpdatePaymentMethod | update credit card info |
+| AuditEventDeletePaymentMethod | delete a credit card |
+| AuditEventCreateAWSVpcPeering | create an aws vpc peering |
+| AuditEventCreateGCPVpcPeering | create a gcp vpc peering | 
+| AuditEventListAWSVpcPeering | list all aws vpc peering in the project |
+| AuditEventListGCPVpcPeering | list all gcp vpc peering in the project |
+| AuditEventDeleteAWSVpcPeering | delete an aws vpc peering |
+| AuditEventDeleteGCPVpcPeering | delete a gcp vpc peering |
+| AuditEventGetProjectTrafficFilter | get traffic filter list of the project |
+| AuditEventUpdateProjectTrafficFilter | update traffic filter list of the project |
+| AuditEventGetTrafficFilter | get traffic filter list of the cluster |
+| AuditEventUpdateTrafficFilter | update traffic filter list of the cluster |
+| AuditEventCreateProjectCIDR | create a new project CIDR |
+| AuditEventGetProjectCIDR | list CIDR in the region |
+| AuditEventGetProjectRegionCIDR | list all CIDR in the project |
+| AuditEventDeleteBackupInRecycleBin | delete backups of deleted clusters in recycle bin |
+| AuditEventChangeClusterRootPassword | reset the root password of cluster |
+| AuditEventCreateImportTask | create an import task |
+| AuditEventCancleImportTask | cancel an import task |
+| AuditEventExitImportTask | exit an import task |
+| AuditEventCreateCluster | create a cluster |
+| AuditEventDeleteCluster | delete a cluster |
+| AuditEventScaleCluster | scale a cluster |
+| AuditEventCreateBackup | make a backup |
+| AuditEventDeleteBackup | delete a backup |
+| AuditEventRestoreBackup | restore from a backup |
+| AuditEventUpdateAuditLogStatus | enable / disable audit logging |
+| AuditEventCreateAuditLogAccessRecord | add audit log filter conditions |
+| AuditEventDeleteAuditLogAccessRecord | delete audit log filter conditions |
+| AuditEventUpdateUserRole | modify user roles |
+
+> **Note:**
+>
+> - Closing the Console audit log feature is not supported;
+> - Setting filter rules is not supported.
+
+## Audit log storage policy
+- Audit log information is currently only supported to be stored in AWS ES;
+- The storage time is 90 days, after which it will be automatically cleaned up.
+
+## View audit logs
+The Console audit log information is temporarily accessible only to internal personnel of TiDB Cloud. Customers who need to view audit log information can seek TiDB Cloud support.
+
+## Audit log fields
+
+The fields in the audit log include: basic fields and extended fields.
+The basic fields are:
+
+| Field name | Data type | Description |
+|---|---|---|
+| timestamp | timestamp | Time of event |
+| Auditeventtype | string | Event type |
+| userID | uint64 | User ID |
+| clientIP | string | Client IP |
+| isSuccess | bool | Event result |
+
+Extended field: supplements audit information based on different event types, helping audit administrators to analyze user activities more accurately.
```

Comment:
Who is the audit administrator? Users or PingCAP personnel?
```suggestion
Extended fields supplement the audit information based on different event types, which helps audit administrators analyze user activities more accurately.
```

Reply:
updated